smartstruct/lib/src/annotations.dart

```diff
@@ -4,8 +4,9 @@
 class Mapper {
   final bool useInjection;
   final bool caseSensitiveFields;
+  final bool static;
```

Comment:
Hey, only thing I'd like to discuss is the name of the configuration. Everything else looks good.
Maybe something like ``generateStaticProxy``?